packages/node/src/generators/application/schema.json

```diff
@@ -77,8 +77,7 @@
     "bundler": {
       "description": "Bundler which is used to package the application",
       "type": "string",
-      "enum": ["esbuild", "webpack"],
-      "default": "esbuild"
+      "enum": ["esbuild", "webpack", "none"]
```

Comment:
Let's add a default value for esbuild. I don't think we'll support "none" right now, but users can always use "bundle: false" with esbuild to transpile only.